mlrun/api/migrations_sqlite/versions/026c947c4487_altering_table_datastore_profiles_2.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""altering table datastore_profiles 2
+
+Revision ID: 026c947c4487
+Revises: 6e0c9531edc7
+Create Date: 2023-08-10 14:15:30.523729
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "026c947c4487"
+down_revision = "6e0c9531edc7"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Don't know if you really need a new alembic migration just for this - you can add the migration to the previous one you added on previous PR and just change it to `object` rather than the `body` column. The only way this will break for someone is if they installed a release that already contained your previous implementation and try to upgrade (i.e. across rc versions). 
It's not terrible to add another migration, just make sure you keep that in mind.